channels/ON-CHAIN.md

```diff
@@ -40,11 +40,12 @@ Serialization defined [here](../serializations.md#channel-create-transaction)
 
 - `initiator`: public key/address of the initiating peer
 - `responder`: public key/address of the responding peer
-- `initiator_amount`: unsigned
-- `responder_amount`:
-- `lock_period`
+- `initiator_amount`: unsigned amount of tokens the initiator commits to the channel
+- `responder_amount`: unsigned amount of tokens the responder commits to the channel
+- `lock_period`: minimal block height interval between a channel_close_solo/last channel_slash transaction and the channel_settle transaction
 - `ttl`
 - `fee`
+- `state_hash`: the root hash of the channel state tree
```

Comment:
"*the* channel state tree". Which channel state tree? Could you please enrich the "Requirements" subsection with the validation rules for this field to be valid/invalid?